package.json

```diff
@@ -87,6 +89,10 @@
     "jest-environment-jsdom": "^29.7.0",
     "lint-staged": "^15.2.7",
     "mini-css-extract-plugin": "^2.9.0",
+    "monaco-editor": "^0.51.0",
```

Comment:
I mentioned in the PR description, but this is a quick note about why we're adding these libs:

* `yaml` – for parsing and formatting the YAML data
* `monaco-editor` – core editor
* `monaco-yaml` – YAML language support and schema validation
* `monaco-editor-webpack-plugin` – proper bundling and worker configuration
* `openapi-types` – typing support for OpenAPI schema parsing